legend-engine-xt-relationalStore-trino-pure/src/main/resources/core_relational_trino/relational/protocols/pure/vX_X_X/extension/metamodel_connection.pure

```diff
@@ -0,0 +1,13 @@
+Class meta::protocols::pure::vX_X_X::metamodel::store::relational::connection::alloy::specification::TrinoDatasourceSpecification extends meta::protocols::pure::vX_X_X::metamodel::store::relational::connection::alloy::specification::DatasourceSpecification
+{
+   host: String[1];
+   port: Integer[1];
+   catalog: String[1];
+   clientTags: String[1];
```

Comment:
Do you want this to be a string with commas or model it as String[0..*]. Also, is it always required ?

Reply:
Let's discuss. Need more info.